<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InstanceManagerDialog</class>
 <widget class="QDialog" name="InstanceManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Instance manager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QWidget" name="widget_8" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QPushButton" name="createNew">
        <property name="text">
         <string>Create new instance</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/MO/gui/add</normaloff>:/MO/gui/add</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="createPortable">
        <property name="text">
         <string>Create portable instance</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/MO/gui/package</normaloff>:/MO/gui/package</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="whatis">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://github.com/ModOrganizer2/modorganizer/wiki/Instance&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;What is an instance?&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1">
     <item>
      <widget class="QWidget" name="widget_4" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QListWidget" name="list"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,1">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_7" native="true">
              <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item row="0" column="1">
                <widget class="QLabel" name="name">
                 <property name="text">
                  <string notr="true">instance name</string>
                 </property>
                 <property name="textInteractionFlags">
                  <set>Qt::TextBrowserInteraction</set>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Game:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Name:</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Location:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QPushButton" name="rename">
                 <property name="text">
                  <string>Rename</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QLabel" name="location">
                 <property name="text">
                  <string notr="true">location path</string>
                 </property>
                 <property name="textInteractionFlags">
                  <set>Qt::TextBrowserInteraction</set>
                 </property>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Base folder:</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QLabel" name="baseDirectory">
                 <property name="text">
                  <string notr="true">base folder path</string>
                 </property>
                 <property name="textInteractionFlags">
                  <set>Qt::TextBrowserInteraction</set>
                 </property>
                </widget>
               </item>
               <item row="5" column="2">
                <widget class="QPushButton" name="exploreBaseDirectory">
                 <property name="text">
                  <string>Explore</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="2">
                <widget class="QPushButton" name="exploreLocation">
                 <property name="text">
                  <string>Explore</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1" colspan="2">
                <widget class="QLabel" name="gameName">
                 <property name="text">
                  <string notr="true">game name</string>
                 </property>
                 <property name="textInteractionFlags">
                  <set>Qt::TextBrowserInteraction</set>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>10</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QWidget" name="widget_9" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="convertToPortable">
                 <property name="text">
                  <string>Convert to portable</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="convertToGlobal">
                 <property name="text">
                  <string>Convert to global</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="deleteInstance">
                 <property name="text">
                  <string>Delete instance</string>
                 </property>
                 <property name="icon">
                  <iconset resource="resources.qrc">
                   <normaloff>:/MO/gui/remove</normaloff>:/MO/gui/remove</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="ok">
        <property name="text">
         <string>Switch to this instance</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/MO/gui/next</normaloff>:/MO/gui/next</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
